package/MDAnalysis/coordinates/PDB.py

```diff
@@ -698,7 +698,7 @@ class PrimitivePDBWriter(base.Writer):
             "ATOM  {serial:5d} {name:<4s}{altLoc:<1s}{resName:<4s}"
             "{chainID:1s}{resSeq:4d}{iCode:1s}"
             "   {pos[0]:8.3f}{pos[1]:8.3f}{pos[2]:8.3f}{occupancy:6.2f}"
-            "{tempFactor:6.2f}      {segID:<4s}{element:>2s}{charge:.2f}\n"),
+            "{tempFactor:6.2f}      {segID:<4s}{element:>2s} {charge:1d}\n"),
```

Comment:
Doesn't that write the charge on a single column (80) rather than 2 (79 and 80)?